src/gcp_scanner/crawl.py

```diff
@@ -562,9 +543,8 @@ def get_bq(project_id: str,
       response = request.execute()
 
       for dataset in response.get("datasets", []):
-        dataset_id = dataset["datasetReference"]["datasetId"]
-        bq_datasets[dataset_id] = get_bq_tables(project_id, dataset_id, service)
-
+        bq_datasets[dataset["datasetReference"]["datasetId"]] = get_bq_tables(project_id, dataset["datasetReference"]["datasetId"], service)
```

Comment:
Removing the temporary variable decreases readability, revert.

Reply:
Hi @ZetaTwo , I have resolved this issue in the latest PR.